### ISSUE TITLE
Use build tag osusergo?

### ISSUE THREAD

**Issue by sb10** [VertebrateResequencing/wr#295]
As of go 1.11, the os/user package can now be built in pure Go mode using the build tag "osusergo", independent of the use of the environment variable CGO_ENABLED=0.

https://golang.org/cmd/go/#hdr-Preliminary_module_support

We want CGO_ENABLED=0 anyway, but is there a benefit to having this build tag as well?

**Comment by TheDiveO** [comment]
yes, because there are situations where you cannot disable cgo but otherwise want to avoid the dependencies on libc. In fact, it is possible to create a static standalone binary with cgo elements and `-ldflags="-extldflags=-static"`. For a real example of where this can be useful, please see [mount namespace sandbox](https://github.com/TheDiveO/lxkns/blob/master/ops/mountineer/mntnssandbox/sandbox.go) in my lxkns project.
